src/test/java/org/openrewrite/java/testing/junit6/MinimumJreConditionsTest.java

```diff
@@ -0,0 +1,1073 @@
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.DisabledOnJre;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  import static org.junit.jupiter.api.condition.JRE.*;
+
+                  class MyTest {
+                      @Test
+                      @DisabledOnJre(%s)
+                      void testNotOnJava17() {
+                          System.out.println("Not Java 17");
+                      }
+                  }
+                  """.formatted(jre)
+              )
+            );
+        }
+
+        @CsvSource({
+          "versions = { 17 },versions = 17",
+          "versions = { 21 },versions = 21",
+        })
+        @ParameterizedTest
+        void unwrapSingleValueArray(String jre, String afterJre) {
+            rewriteRun(
+              java(
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.DisabledOnJre;
+
+                  class MyTest {
+                      @Test
+                      @DisabledOnJre(%s)
+                      void notOnThisJava() {
+                          System.out.println("Not this Java");
+                      }
+                  }
+                  """.formatted(jre),
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.DisabledOnJre;
+
+                  class MyTest {
+                      @Test
+                      @DisabledOnJre(%s)
+                      void notOnThisJava() {
+                          System.out.println("Not this Java");
+                      }
+                  }
+                  """.formatted(afterJre)
+              )
+            );
+        }
+    }
+
+    @Nested
+    class EnabledForJreRange {
+        @ParameterizedTest
+        @ValueSource(strings = {
+          "min = JRE.JAVA_8, max = JRE.JAVA_11",
+          "minVersion = 8, maxVersion = 11"
+        })
+        void removeTestEnabledForRangeEndingBeforeMinimum(String range) {
+            rewriteRun(
+              java(
+                """
+                  import org.junit.jupiter.api.Test;
+                  import org.junit.jupiter.api.condition.EnabledForJreRange;
+                  import org.junit.jupiter.api.condition.JRE;
+
+                  class MyTest {
+                      @Test
+                      @EnabledForJreRange(%s)
+                      void testOnJava8To11() {
+                          System.out.println("Java 8-11");
+                      }
+                  }
+                  """.formatted(range),
+                """
+                  class MyTest {
+                  }
+                  """
+              )
+            );
+        }
+
+        @ParameterizedTest
+        @ValueSource(strings = {
+          "min = JRE.JAVA_17, max = JRE.JAVA_21",
+          "minVersion = 17, maxVersion = 21"
```

Comment:
Would it make sense here to remove the minimum version? Before JUnit 6 that had a signal, now that's implied from running on JUnit 6.